### PR TITLE
Vanilla Slat Caged APCs

### PR DESCRIPTION
**Pull request by ante185**
## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [ ] Enhancement

### What have you changed and why?
Information:
Patched the vanilla APCs to spawn with **Slat cages**, increasing the number of hits that they can take.
Moved the AFV Rhino down to light tank category.
Moved the IFV Mora and Kamysh up to IFV category.

**Reduced or removed the overlap between militia and military trucks of the vanilla templates.**

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:

### CODE REVIEW

**Review by Lazejun — COMMENTED** [review]
No idea what's Bob's and John's opinion but Fucking around with Vanilla Classes is the wrong approach.

This could cause funny problems for people that load Antistasi + it messes with mods that Base assets on the base Class.

The safer approach is create a new Class based on the Base one and add shit to the new class

**Comment by jaj22** [comment]
Makes me wonder if I've been reading previous patches correctly (ok I mostly left them to Bob), but yes: Add new version with slats enabled. Then maybe use both in the templates in different places.

**Comment by ante185** [comment]
Updated

File: A3A/addons/config_fixes/Vanilla/CfgVehicles.hpp [hunk 1]
```diff
@@ -3,6 +3,7 @@
 class CfgVehicles 
 {
 #include "air.hpp"
+#include "ifv.hpp"
```
Comment (Bob-Murphy) [inline A3A/addons/config_fixes/Vanilla/CfgVehicles.hpp:6]: please put into alphabetical order

File: A3A/addons/core/Templates/Templates/Vanilla/Vanilla_AI_AAF.sqf [hunk 2]
```diff
@@ -56,7 +56,7 @@ private _AA = ["I_LT_01_AA_F"];
 
 //Config special vehicles - militia vehicles are mostly used in the early game, police cars are being used by troops around cities -- Example:
 private _vehiclesMilitiaLightArmed = ["a3a_Offroad_01_green_armed_F", "a3a_Offroad_01_green_AT_F"];
-["vehiclesMilitiaTrucks", ["I_Truck_02_transport_F", "I_Truck_02_covered_F"]] call _fnc_saveToTemplate;
+["vehiclesMilitiaTrucks", ["I_C_Van_01_transport_F"]] call _fnc_saveToTemplate;
```
Comment (Bob-Murphy) [inline A3A/addons/core/Templates/Templates/Vanilla/Vanilla_AI_AAF.sqf:59]: Revert please.
Van is Laws of War DLC content.

File: A3A/addons/core/Templates/Templates/Vanilla/Vanilla_AI_LDF.sqf [hunk 3]
```diff
@@ -56,7 +56,7 @@ private _HelisTransport = ["B_Heli_Transport_01_camo_F"];
 
 //Config special vehicles
 private _vehiclesMilitiaLightArmed = ["a3a_Offroad_01_ldf_armed_F", "a3a_Offroad_01_ldf_AT_F"];
-["vehiclesMilitiaTrucks", ["I_E_Truck_02_F", "I_E_Truck_02_transport_F"]] call _fnc_saveToTemplate;
+["vehiclesMilitiaTrucks", ["I_E_Van_02_transport_F", "I_E_Van_02_vehicle_F"]] call _fnc_saveToTemplate;
```
Comment (Bob-Murphy) [inline A3A/addons/core/Templates/Templates/Vanilla/Vanilla_AI_LDF.sqf:59]: Revert please.
Van is Laws of War DLC content.

**Review by Bob-Murphy — CHANGES_REQUESTED** [review]

File: A3A/addons/core/Templates/Templates/Vanilla/Vanilla_AI_AAF.sqf (hunk 2 above)
Comment (ante185) [reply A3A/addons/core/Templates/Templates/Vanilla/Vanilla_AI_AAF.sqf:59]: I_C_Van_01_transport_F, you're thinking of Van_02

File: A3A/addons/core/Templates/Templates/Vanilla/Vanilla_AI_LDF.sqf (hunk 3 above)
Comment (ante185) [reply A3A/addons/core/Templates/Templates/Vanilla/Vanilla_AI_LDF.sqf]: Resolved.

**Review by Bob-Murphy — APPROVED** [review]
lgtm

**Pull request merged**